packages/contracts/contracts/discovery/Batch.sol

```diff
@@ -43,7 +43,7 @@ contract Batch is IBatch, ICommon, ProjectVoting {
     }
 
     constructor(address[] memory _projects, uint256 _slotCount)
-        ProjectVoting(_projects)
+        ProjectVoting(_projects, _slotCount)
```

Comment:
if this is, as stated in the description, suposed to equal slot count, then how about instead of adding a new storage variable, creating a virtual function of the same name on ProjectVoting, and implement it here?

```
function _voteLimit() internal override(ProjectVoting) {
  return slotCount
}
```

in general, the less storage the better

Reply:
You mean moving that check to Batch instead of the ProjectVoting? Doesn't that break the abstraction a bit?

Reply:
not moving the check, just turning `voteLimit` from a storage value into a virtual function that the subclass is forced to implement